packages/mail/templates/membership_owner_autopay_successful.html

```diff
@@ -43,11 +43,15 @@
                   <td style="padding: 20px">
                     <p>Guten Tag</p>
                     <p>
-                      Wir haben Ihre Mitgliedschaft automatisch um ein weiteres
-                      Jahr verlängert und dafür Ihre {{autopay_card_brand}} mit
-                      den Endziffern {{autopay_card_last4}} mit
-                      {{autopay_total}} belastet. Wir bedanken uns für Ihre
-                      Treue und Ihr Vertrauen.
+                      Wie bereits am {{end_date_minus_10}} angekündigt, haben
```

Comment:
Depending on state of our infrastructure – if schedulers are running or paused for some reason – this date can vary. Better to use a relative term, like «Wie bereits vor einigen Tagen…».